scripts/set_retroactive_cleaned_at.py

```diff
@@ -0,0 +1,35 @@
+import click
+import datetime as dt
+import yaml
+from pathlib import Path
+from cg.store import Store, models
+
+
+@click.command("update-cleaned-status")
+@click.argument("config_path")
+@click.pass_context
+def set_analysis_status_cleaned_mip(config_path: str):
+    """Updates all cleaned_at timestamps for mip cases"""
+
+    config_dict = yaml.safe_load(open(config_path))
+    case_dir = config_dict["mip-rd-dna"]["root"]
+    status_db = Store(config_dict["database"])
+
+    # Fetch all mip cases
+    relevant_cases = status_db.query(models.Family).filter(
+        models.Family.data_analysis.ilike("%mip%")
```

Comment:
This could possible include RNA, is that intentional?

Reply:
Its intentional - MIP dna and MIP rna cases are stored/cleaned the same way